_docs/runtime/git-sources.md

```diff
@@ -43,9 +50,43 @@ Git Sources are displayed when you drill down on a runtime.
     max-width="30%" 
 %}
 
-{:start="3"}
-1. If you have already installed the Codefresh CLI, select **I already have...**, and continue with Git Source creation in the lower part of the panel.
-  >If you created a new GitOps repo for the Git Source, make sure that the Git token has a valid expiration date, `scopes` (`repo` and `admin-repo.hook`), with `base64` encoding.  
+  * **Git Source Name**: The name of the Git Source, which must be unique within the cluster.
+  * **Source**: The Git repo with the desired state, tracked by the Git Source, and synced to the destination cluster.  
+    * **Repository**: Mandatory. The URL to the Git repo.  
+    * **Branch**: Optional. The specific branch within the repo to track.  
+    * **Path**: Optional. The specific path within the repo, and branch, if one is specified, to track.  
+  * **Destination**: The destination cluster with the actual state to which to apply the changes from the **Source**.  
+    * **Namespace**: The namespace in the destination cluster to which to sync the changes.  
+
+  * **Include Files** and **Exclude Files**: The file or files to include or exclude from the Git repo when syncing to the destination cluster.  
+    Use GLOB to define patterns using wildcards to match path names in the source Git repo.  
+	For example, `workflows/**/*.yaml`, in the Include Files field would include all files in the `workflows` directory and all its child directories, with `.yaml` as the extension.  
+    `**/images/**/*` in the Exclude Files field, would ignore all directories entitled `images`.  
+	For GLOB guidelines and examples, see this [article](https://deepsource.io/blog/glob-file-patterns/){:target="\_blank"}.
+
+{:start="4"}
+1. Select **+ Create Git Source**.
+
+### Edit Git Source definitions
+Edit an existing Git Source by changing the source and destination definitions.  
+> You cannot change the name of the Git Source.
+
+1. In the Codefresh UI, go to the [Runtimes](https://g.codefresh.io/2.0/account-settings/runtimes**){:target="\_blank"} page.
+1. From the **List View** (the default), select the runtime with the Git Source, and then select the **Git Sources** tab.  
+1. In the row with the Git Source to edit, select the three dots, and then select **Edit** in the panel that appears.
+
+{% include 
+	image.html 
+	lightbox="true" 
+	file="/images/runtime/edit-git-source.png" 
+	url="/images/runtime/edit-git-source.png" 
+	alt="Edit Git Source" 
+	caption="Edit Git Source"
+    max-width="30%" 
+%}
+{:start="4"}
+1. Change the **Source** and **Destination** definitions for the Git Source, and select **Save**. 
 
 ### What to read next
```

Comment:
What do I do with the git source after it is created? Where can I use it?

Reply:
Once you create a Git Source for a runtime, it is available for selection to store resources for a CI/CD entity assocaited with that runtime . For example, when you create pipelines or applications, you can select the Git Source in which to store manifest definitions.